### ISSUE TITLE
Il tasto "Carica altri risultati" carica anche le bozze 

### ISSUE THREAD

**Issue by Cam-al** [italia/design-comuni-wordpress-theme#403]
### Prerequisiti

- [X] Ho già [cercato](https://github.com/italia/design-comuni-wordpress-theme/issues?utf8=%E2%9C%93&q=is%3Aissue) tra le issue la presenza di un problema simile
- [X] Ho letto le [linee guida di contribuzione](https://github.com/italia/design-comuni-wordpress-theme/blob/main/CONTRIBUTING.md)

### Descrivi il problema

Ho iniziato a inserire circa 30 servizi ma al momento solo 7 sono completi e possono essere pubblicati nel sito del comune, quindi tutti gli altri servizi che ho iniziato a caricare ma che non sono ancora disponibili nel comune gli ho messi in bozza per non mostrali nell'elenco. Tutti quelli in bozza giustamente non vengono mostrati, ma vengono "contati" quando si clicca carica altri risultati.
esempio:
nella pagina "Servizi" vedo i primi tre servizi in ordine alfabetico del titolo
se clicco su carica altri risultati mostra solo un servizio 
se clicco una seconda volta mostra sempre e solo lo stesso
se clicco altre 3 volte aggiunge due servizi
questo perché tra il primo servizio pubblicato e gli altri due ci sono circa 10 servizi in bozza.

non so se è chiaro, ma in poche parole se un servizio nell'elenco di tutti i servizi ordinati per titolo è tredicesimo verrà mostrato comunque in tredicesima posizione anche se 10 di quelli precedenti sono bozze non visibili.

### Su quale sisitema operativo stai riscontrando il problema?

Windows

### Su quale browser stai riscontrando il problema?

Qualsiasi

### Che versione del tema stai utilizzando?

1.7.4

**Comment by enrimk** [comment]
Vero: [_load_more()_ ora carica i post in tutti gli stati](https://github.com/italia/design-comuni-wordpress-theme/blob/c88ddf906164b612b0edcf14bca4675f0d5d2e24/inc/load_more.php#L38-L54), non solo quelli pubblicati - e non solo per i servizi, anche per gli altri tipi di contenuto.
Poi, [i servizi vengono filtrati per status solo "a posteriori"](https://github.com/italia/design-comuni-wordpress-theme/blob/c88ddf906164b612b0edcf14bca4675f0d5d2e24/template-parts/servizio/card.php#L8), al momento di visualizzarli, scartando quelli in bozza; ma non è un buon modo di procedere. 

È vero che omettendo il parametro `post_status` [il default di WP_Query è "publish"](https://github.com/WordPress/WordPress/blob/2f4ffff0fbf64ac043dddadcd3f73b2c1ffa9c0d/wp-includes/class-wp-query.php#L2660) - ma solo per il fronte-sito.
[Per il backoffice, le cose cambiano](https://github.com/WordPress/WordPress/blob/2f4ffff0fbf64ac043dddadcd3f73b2c1ffa9c0d/wp-includes/class-wp-query.php#L2668-L2678) (nelle liste di admin devono pur esserci anche le bozze).
E le chiamate ajax ad _admin-ajax.php_ contano come backoffice ([_is_admin()_ è _true_](https://github.com/WordPress/WordPress/blob/2f4ffff0fbf64ac043dddadcd3f73b2c1ffa9c0d/wp-admin/admin-ajax.php#L18)).

Soluzione: aggiungere esplicitamente `'post_status' => 'publish'` [negli argomenti della query di _load_more()_](https://github.com/italia/design-comuni-wordpress-theme/blob/c88ddf906164b612b0edcf14bca4675f0d5d2e24/inc/load_more.php#L38-L54).

**Comment by Cam-al** [comment]
Si ha funzionato, grazie.
L'ho modificato cosi:
`$args = array(
        's' => $_POST['search'],
        'posts_per_page' => $_POST['post_count'] + $_POST['load_posts'],
        'post_type'      => $post_types,
        'orderby'        => 'date',
        'order'          => 'DESC',
	'post_status' => 'publish'
    );
	
	if ( $post_types != "notizia" && $post_types != "evento" ) {
		$args = array(
			's' => $_POST['search'],
	    'posts_per_page' => $_POST['post_count'] + $_POST['load_posts'],
	    'post_type'      => $post_types,
			'orderby' => 'post_title',
			'order'   => 'ASC',
			'post_status' => 'publish'
		);
	}`

Ho aggiunto anche gli eventi in ordine di data

**Comment by enrimk** [comment]
Per l'ordinamento degli eventi, attenzione però che l'ordinamento di default è sì per data, ma per data di _pubblicazione_ (a meno che tu non abbia aggiunto altre eccezioni non riportate qui).
Un evento si dovrebbe ordinare per data _di inizio_.
D'altra parte mi pare di aver visto anche in altre parti del tema questa confusione tra date. Nel calendario in homepage, forse?

**Comment by enrimk** [comment]
Issue risolto nella 1.10.0, da [una modifica non documentata](https://github.com/italia/design-comuni-wordpress-theme/pull/434/files?diff=unified&w=1#diff-4cb36dfa1f7943c9e68291dbe05df4580bfbbe05c5c9a4c645a9c24291f01b8e), ma integrata "clandestinamente" 😃 con la #434 .

**Comment by zetareticoli** [comment]
@Cam-al posso chiudere quindi la issue?

**Comment by Cam-al** [comment]
si
